src/control/cam_control_mod.meta

```diff
@@ -0,0 +1,32 @@
+[ccpp-table-properties]
+  name = cam_control_mod
+  type = module
+
+[ccpp-arg-table]
+  name  = cam_control_mod
+  type  = module
+[ aqua_planet ]
+  standard_name = do_aqua_planet
+  units = 1
+  type = logical
+  dimensions = ()
+[ eccen ]
+  standard_name = eccentricity_factor
+  units = 1
+  type = real | kind = r8
+  dimensions = ()
+[ obliqr ]
+  standard_name = obliquity
+  units = radians
+  type = real | kind = r8
+  dimensions = ()
+[ lambm0 ]
+  standard_name = mean_longitude_of_perihelion_at_vernal_equinox
+  units = radians
+  type = real | kind = r8
+  dimensions = ()
+[ mvelpp ]
+  standard_name = moving_vernal_equinox_longitude_perihelion_plus_pi
```

Comment:
I might change this standard name slightly:

```suggestion
  standard_name = moving_vernal_equinox_longitude_of_perihelion_plus_pi
```

Reply:
changed!